quickstarts/microsoft.machinelearningservices/machine-learning-create-vnet/GettingStarted.md

```diff
@@ -0,0 +1,238 @@
+
+### Using Azure CLI with Visual Code
+
+In the terminal window type: **az login**
+
+![Screen](./images/azlogin.png)
+
+You will be redirected to the Azure Portal in your web browser where you can insert your credentials and log in.
+
+After logging in, you will see your credentials on the terminal.
+
+To set the right subscription, type following command:
+
+#### az account set --subscription "your subscription id"
+
+![Screen](./images/azsetsub.png)
+
+### Resource Group
+
+Now you need a Resource Group for our deployment. If you haven't already created a Resource Group, you can do it now. If you are new to Azure and wonder what is a Resource Group? Bare with me! A Resource Group is a container that holds related resources for an Azure solution. The resource group includes those resources that you want to manage as a group. Simply saying: it's like a folder that contains files. Simple as that.
+
+To create a Resource Group, you need a name and a location for your Resource Group.
+
+For a list of locations, type: **az account list-locations**
+
+The Machine Learning service isn't yet available for all locations. Check the availability of locations [here](https://azure.microsoft.com/en-gb/global-infrastructure/services/?products=).
+
+To create the Resource group, type the command:
+
+#### az group create --name "resource-group" --location "your location"
+
+![Screen](./images/azgroup.png)
+
+Super simple, right? Now that we have our **Resource Group** created, let's deploy the **Azure Machine Learning workspace** using the az CLI.
+
+#### az group deployment create --name "name of your deployment" --resource-group "resource-group" --template-file "./azuredeploy.json"
+
+As you can see, it's running.
+
+![Screen](./images/azdeploy.png)
+
+Go grab a cup of coffee, have some fresh air. Before you come back you will have your **Azure Machine Learning workspace**.
+
+And there we go, your deployment has Succeeded:
+
+![Screen](./images/azdeploy2.png)
+
+Let's check the resource in the [Azure Portal](https://portal.azure.com).
+
+On the portal, navigate to Resource Groups. On this blade, you can see the Resource Group we've created.
+
+Click on the Resource Group to expand and show the **Resources**:
+
+- Application Insights
+- Key vault
+- Container registry
+- Storage account
+- Machine Learning
+
+![Screen](./images/azdeployportal.png)
+
+Click on **Machine Learning** and you will have an overview of your **WorkSpace**.
+
+![Screen](./images/azdeployportal2.png)
+
+Congratulations, you have deployed a **Machine Learning Workspace** and that is just the tip of the Iceberg.
+
+Most important, don't forget to have fun!
+
+### Using the Portal
+
+Using your favorite web browser Navigate to the Portal, in All Services look for **Templates**, you may want to add this service to favorites.
```

Comment:
this feature is being deprecated, you can remove this or replace it with instructions for templateSpecs